src/universal/components/StyledFontAwesome.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import FontAwesome from 'react-fontawesome';
+
+const StyledFontAwesome = (props) => {
```

Comment:
@ackernaut you may wanna use this for FontAwesome components that have conditional logic.

Reply:
👍